RPIPs/RPIP-vop.md

```diff
@@ -0,0 +1,309 @@
+author: LongForWisdom (@LongForWisdom)
+discussions-to: 
+status: Draft
+type: Meta
+created: 2023-10-02
+---
+
+## Abstract
+Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
+
+$TODO
+
+## Motivation
+
+The motivation for some VOP-like framework can be summarized as the desire for increased transparency and extensibility for DAO operational decision-making.
+
+**Transparency**
+
+A VOP-like framework allows participants in the Rocket Pool DAOs to more easily be able to understand the common processes that shape the DAOs. It does this in a few main ways:
+* The types of process that exist can be listed, summarized and presented in such a way that is more-approachable to the average participant than reading through multiple full RPIPs. Participant attention is a key resource and should not be wasted on implementation details, if all that is desired is a high-level overview of 'how does process x work in Rocket Pool'.
+* Votes and discussions can be labelled explicitly according to the process they are using. This makes it easier for participants to quickly slot items into the appropriate mental boxes. An RPIP is important, because it represents a change to the status quo. A GMC election is also important, but it doesn't represent a change to how Rocket Pool works. A grants appeal is potentially unimportant to a less-engaged participant, and can be more safely ignored. 
+* The outcomes of processes can be recorded in a clear, consistent and accessible manner. Again, the goal is to minimize the attention and mental effort required by potential participants to find information they are looking for.
+
+**Extensibility**
+
+Adoption of a VOP-like framework helps to structure the processes of the Rocket Pool DAO in such a way that is easily extensible in future improvement proposals. The motivation here is to encourage organization, transparency and accessibility in such a way that it becomes self-sustaining. If there is a clear existing structure in use, then it encourages its use in future RPIPs.
+
+## Specification
+$TODO Remove
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+### VOP Definition
+This section describes what a VOP is, and when to use the VOP framework.
+
+* A Voted Operational Process (VOP) is a process involving a vote through which an operational decision is made.
+  * Some operational decisions within the Rocket Pool DAOs MAY NOT require a vote, and therefore are not included in the VOP framework.
+* A VOP MUST NOT be defined to cover decisions that are improvements to the Rocket Pool protocol, Rocket Pool DAOs, or anything that should otherwise require use of the RPIP process.
```

Comment:
The "improvements" piece is potentially challenging here. Why would we be adopting a VOP if it did not improve the protocol state in some way?

My reading is the intention here is that VOPs would just be formalizing informal procedures,  or otherwise offering a clear pathway to addressing something that is a frequent community concern. If that is correct, perhaps its better to avoid using "improvements" and instead focus on the fundamentals of when something is a VOP framework and when it is an RPIP. 

I think the line is probably closer to when something is a new challenge or direction it should be an RPIP and when it is aggregating and streamlining an established process it should be a VOP, but would like to check my understanding given my initial misfire here

Reply:
Adopting the VOP is an improvement, and is fine. The instances of the VOP should not represent improvements. Will try to clarify.